entrypoint.sh

```
@@ -3,13 +3,26 @@
+
 echo "Generating config.json from template..."
 
 if ! envsubst < /usr/share/nginx/html/config.template.json > /usr/share/nginx/html/config.json; then
-    echo "Error: Failed to generate config.json from template"
+    echo "Error: Failed to generate config.json from template" >&2
+    exit 1
+fi
+
+echo "Generating nginx.conf from template..."
+
+if ! envsubst '${STATIC_CACHE_CONTROL}' < /etc/nginx/nginx.conf.template > /tmp/nginx.conf; then
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)